### DISCUSSION TITLE
#32 target | mix-blend-mode doesnt count right way

### DISCUSSION THREAD

**Discussion by StoneLard** [cssbattle/forum#210] [Bugs reports]
[code 2.txt](https://github.com/cssbattle/forum/files/7822669/code.2.txt)
#32 target calculates only around 80%
(center sqr color in photoshop same as task color)

**Comment by SelenIT** [comment]
It's really strange, but apparently `background: #FFFFFF;` of `body` doesn't work well with `mix-blend-mode` on the server (I can't explain it, likely it's just a bug). You can just drop it, default white background of the page seems enough here.

**Reply by gunnarbittersmann** [reply]
I’ve tried adding `body{background:#ffffff}` to some working solutions.
It reduces the match to 94% in my cases where the arms of the cross get their colors from `border-color` or `outline-color`.
But it’s still a 100% match when they get their colors from `color`.
Really strange, indeed.

**Reply by StoneLard** [reply]
thx for answers, glad im not insane))

**Comment by StoneLard** [comment]
yeah, just have tried without ` body{background:#ffffff}` and got 613 points

**Reply by gunnarbittersmann** [reply]
If you want to get more:
- remove all unnessesary spaces (all but the ones between element type and attribute) and line breaks
- remove trailing `;` in every declaration block
- remove `</style>` end tag
- never use `div` (nor `span`)
- never use `class` (use single-character attribute names and attribute selectors instead)

**Reply by StoneLard** [reply]
yeah, i understand how it works, my goal is 100% match with normal code structure, thx for the advice anyway

**Comment by StoneLard** [comment]
`background: #FFFFFF; isolation: isolate;` works well too btw
